ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -508,8 +508,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     new Site {SiteId = (int)SiteId.CRYPTIC, Description = "Cryptic disseminated"},
                     new Site {SiteId = (int)SiteId.GASTROINTESTINAL, Description = "Gastrointestinal/peritoneal"},
                     new Site {SiteId = (int)SiteId.GENITOURINARY, Description = "Genitourinary"},
-                    new Site {SiteId = (int)SiteId.LYMPH_INTRA, Description = "Intra-thoracic"},
-                    new Site {SiteId = (int)SiteId.LYMPH_EXTRA, Description = "Extra-thoracic"},
+                    new Site {SiteId = (int)SiteId.LYMPH_INTRA, Description = "Lymph nodes: Intra-thoracic"},
+                    new Site {SiteId = (int)SiteId.LYMPH_EXTRA, Description = "Lymph nodes: Extra-thoracic"},
```

Comment:
Ah, right - so because we are changing the database-defining file, we'll also need to create a migration file (using the `dotnet ef migrations add TweakDiseaseSiteDescriptions -c ntbsContext` invocation).

I'll run that on this branch soon (I don't think it's worth you installing the `dotnet` tools right now, though do feel free to!)